domainobjs/ts/ballot.ts

```diff
@@ -0,0 +1,156 @@
+import * as assert from "assert";
+import {
+    genRandomSalt,
+    hash5,
+    hashLeftRight,
+    IncrementalQuinTree,
+} from "maci-crypto";
+
+/**
+ * A Ballot represents a User's votes in a Poll, as well as their next valid
+ * nonce.
+ * @param _voiceCreditBalance The user's voice credit balance
```

Comment:
Noob question - what do params of a Class represent?

I figured it was describing the parameters that the constructor expects when creating an instance of the Ballot class, but this appears to not be the case

Reply:
I believe this is a repetition, I have removed them. Seems like the norm is to describe the parameters on the constructor